### ISSUE TITLE
[Accessibility] [Code] Incorrect HTML language

### ISSUE THREAD

**Issue by ceciliabelem** [cambiatus/frontend#857]
**What we have**
When access [muda’s page](https://muda.cambiatus.io/join?redirect=%2Fdashboard), the html language should be in portuguese or change as the user changes the navigation language. It impacts the screen readers behaviour.
--

**Proposal**
If the language is portuguese, the html must be “pt-br” too. In this way, the screen reader verbalizes as native. The same behaviour must work for all languages

**WCAG Criteria**
3.1.1

**Platform**
Windows - probably also impacts mobile

Criticity: **
High [A]
